src/main/g8/$name__norm$-stream-impl/src/main/resources/application.conf

```diff
@@ -27,3 +27,12 @@ lagom.persistence.read-side.cassandra.keyspace = \${$name;format="norm"$-stream.
 # Prefer 'ddata' over 'persistence' to share cluster sharding state for new projects.
 # See https://doc.akka.io/docs/akka/current/cluster-sharding.html#distributed-data-vs-persistence-mode
 akka.cluster.sharding.state-store-mode = ddata
+
+# Enable the serializer provided in Akka 2.5.8+ for akka.Done and other internal
+# messages to avoid the use of Java serialization.
+akka.actor.serialization-bindings {
```

Comment:
seems that we've never add it on the stream service

Reply:
The stream service doesn't need because it doesn't use a persistent entity (therefore doesn't serialise any message across the cluster)

Reply:
It does import Lagom Persistence though, but probably only to have access to Cassandra.
https://github.com/lagom/lagom-java.g8/blob/1.5.x/src/main/g8/build.sbt#L47

On the other hand, this is not only about persistence. Actually, the types here are about inter-note serialization, `Done`, `NotUsed`, etc.

Maybe not relevant for the scope of hello world app.